ui-tests/src/test/java/org/openmrs/reference/RecordMergeIssueTest.java

```diff
@@ -41,9 +43,10 @@ public void setUp() throws Exception {
         patient1 = new TestPatient();
     }
 
-    @Ignore
+    @Test
+    @Category(BuildTests.class)
     public void recordMergeIssueTest() throws Exception {
-        homePage.goToRegisterPatientApp();
+        homePage.goToRegisterPatientApp().waitForPage();
```

Comment:
After the test invoking many pages during the test `setUp()`, the patient registration  page takes a bit longer to be loaded  by the `classLoader`. Adding `waitforPage()` resolves the delay **Otherwise** the actions that follow are not effective and the patient data is not saved in `db instance`